core/templates/pages/release-coordinator-page/navbar/release-coordinator-navbar.component.html

```diff
@@ -0,0 +1,174 @@
+          <picture>
+            <source type="image/webp" [srcset]="getStaticImageUrl('/logo/288x128_logo_white.webp')">
+            <source type="image/png" [srcset]="getStaticImageUrl('/logo/288x128_logo_white.png')">
+            <img [src]="getStaticImageUrl('/logo/288x128_logo_white.png')"
+                 class="oppia-logo oppia-logo-wide"
+                 alt="Oppia Logo">
+          </picture>
+          Return to Oppia Library Page
+        </a>
+        <ul class="nav navbar-nav oppia-navbar-breadcrumb">
+          <li>
+            <span class="oppia-navbar-breadcrumb-separator"></span>Release coordinator
+          </li>
+        </ul>
+      </div>
+      <div class="navbar-header float-right">
+        <ul class="nav oppia-navbar-nav oppia-navbar-profile-release-coordinator">
+          <li>
+            <ul class="nav oppia-navbar-tabs-release-coordinator oppia-hide-in-small-width-screens">
+              <li class="nav-item oppia-clickable-navbar-element float-right">
+                <a class="nav-link oppia-navbar-tab"
+                   (click)="switchTab(TAB_ID_JOBS)"
+                   href="#"
+                   [ngClass]="{ 'active': activeTab == TAB_ID_JOBS }"
+                   ngbTooltip="Jobs"
+                   placement="bottom"
+                   container="body">
+                  Jobs
+                </a>
+              </li>
+              <li class="nav-item oppia-clickable-navbar-element float-right">
+                <a class="nav-link oppia-navbar-tab"
+                   (click)="switchTab(TAB_ID_MISC)"
+                   href="#"
+                   [ngClass]="{ 'active': activeTab == TAB_ID_MISC }"
+                   ngbTooltip="Miscellaneous"
+                   placement="bottom"
+                   container="body">
+                  Misc
+                </a>
+              </li>
+            </ul>
+          </li>
+          <li>
+            <ul class="nav oppia-navbar-tabs-release-coordinator">
+              <li ngbDropdown class="nav-item dropdown float-right" [ngClass]="{'open' : profileDropdownIsActive}">
+                <a ngbDropdownToggle class="nav-link oppia-navbar-dropdown-toggle"
+                   data-toggle="dropdown"
+                   (mouseenter)="activateProfileDropdown()"
+                   (mouseleave)="deactivateProfileDropdown()">
+                  <div class="oppia-navbar-profile-picture-container dropdown-toggle float-left">
+                    <span *ngIf="profilePictureDataUrl">
+                      <!--
+                        Lighthouse flags <img> elements that don't have alt attributes:
+                        If the image acts as decoration and does not provide any useful content, give it an empty alt=""
+                        https://web.dev/image-alt/?utm_source=lighthouse&utm_medium=devtools#how-to-add-alternative-text-to-images
+                      -->
```

Comment:
Can you limit this to 80 (or maybe even just 100) chars per line?

Reply:
https links are not expected to be broken down into multiple lines. (make it hard to write and use the link) [Also, the lint checks allow this for the same reason, let me know what you think?]

Reply:
Right, but I'm not talking only about the URL here but also the rest of the comment.

Reply:
![image](https://user-images.githubusercontent.com/16653571/119743227-09e8ce00-bea7-11eb-8d14-9e728f357781.png)

It's around the same length as the above tags and looks kinda nice to me, I'm fine with changing it, let me know if think I should divide the 2nd line? [Note: This code also exists in admin-page-template file in the same pattern]